pydy/viz/scene.py

```diff
@@ -154,6 +175,146 @@ def reference_frame(self, new_reference_frame):
         else:
             self._reference_frame = new_reference_frame
 
+    @property
+    def system(self):
+        return self._system
+
+    @system.setter
+    def system(self, new_system):
+
+        if new_system is not None and not isinstance(new_system, System):
```

Comment:
Can you please review #242 instead of this , I have added some extra things to this PR.